common/src/main/java/hardcorequesting/common/client/ClientChange.java

```diff
@@ -167,11 +166,29 @@ public IMessage build(Object data) {
             return null;
         }
     }
-
+    
     public interface ClientUpdater<T> {
-
+        
         IMessage build(T data) throws IOException;
-
-        void parse(EntityPlayer player, String data);
+        
+        void parse(Player player, String data);
+    }
+    
+    public static class Tuple<A, B> {
```

Comment:
Why do we need this? There is a Pair class from apache than achieves exactly this